twitterpuller/src/main/java/de/brandwatch/minianalytics/twitterpuller/TwitterPullerApplication.java

```diff
@@ -15,12 +15,18 @@ public static void main(String[] args) {
         SpringApplication.run(TwitterPullerApplication.class, args);
     }
 
+    private TweetFetcher tweetFetcher;
+
+    @Autowired
+    Producer producer;
```

Comment:
This can be private. Also I would recommend to get into the habit of using constructor based Autowiring instead, which is generally [recommended](https://kinbiko.com/java/dependency-injection-patterns/). Then it can also be `final`.